proto/rill/runtime/v1/resources.proto

```diff
@@ -153,6 +153,17 @@ message MetricsViewSpec {
     repeated FieldConditionV2 include = 3;
     repeated FieldConditionV2 exclude = 4;
   }
+  enum DefaultComparisonMode {
+    DEFAULT_COMPARISON_MODE_UNSPECIFIED = 0;
+    DEFAULT_COMPARISON_MODE_NONE = 1;
+    DEFAULT_COMPARISON_MODE_TIME = 2;
+    DEFAULT_COMPARISON_MODE_DIMENSION = 3;
+  }
+  // DefaultComparison settings for comparison controls.
+  message DefaultComparison {
+    DefaultComparisonMode mode = 1;
+    string dimension = 2;
+  }
```

Comment:
1. Can we inline this to `default_comparison_mode` and `default_comparison_dimension` in the spec (not the YAML). Generally the protobuf definitions are fine to have more flattened than the YAML (keeps the types easier to navigate)
2. What do you think about dropping the `mode` and just setting `dimension` to the time dimension name if you want to compare by time?

Reply:
1. Makes sense.
2. There might be more of these modes. Also the time dimension should match the `time_dimension` otherwise the UI wont work as expected, so it is better to have a single source for the `time_dimension` and comparison only mention that it is a time comparison.